src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/AndroidUpdateResourcesTest.cs

```diff
@@ -556,7 +556,7 @@ public void CheckOldResourceDesignerIsNotUsed ([Values (true, false)] bool isRel
 			};
 			proj.SetProperty ("AndroidUseIntermediateDesignerFile", "True");
 			proj.SetProperty ("AndroidUseManagedDesignTimeResourceGenerator", "False");
-			using (var b = CreateApkBuilder (Path.Combine ("temp", TestContext.CurrentContext.Test.Name))) {
+			using (var b = CreateApkBuilder (Path.Combine ("temp", TestName))) {
```

Comment:
For these, should we just do:
```suggestion
			using (var b = CreateApkBuilder ()) {
```